### PR TITLE
pointer-events  support versions

### PR DESCRIPTION
**Pull request by Ahmad-Abo-Saa**
I tested pointer-events on following browsers and it worked with me
chrome_android: 71
edge_mobile: 42
firefox_android: 64
opera_android: 49
safari_ios : 12

### CODE REVIEW

**Comment by ddbeck** [comment]
Hi @Ahmad-Abo-Saa and welcome to BCD! Thank you for opening this PR.

First a general note: we try to substantiate `"version_added"` numbers as the version when that feature was first available, not the earliest known version (as in many cases, testing would reveal that the current version works, but give us no information about how far back that support goes). If we know a feature is supported but not which version it was first added, then `true` is the preferred value.

On this feature specifically, could you share a little more detail about how you tested this feature? I'm not quite sure how that would work for devices without a mouse.

**Comment by Ahmad-Abo-Saa** [comment]
Hi @ddbeck, Thanks for your check.
I updated the PR as you suggested to set true instead of version numbers, as you said, I just used this property and tested it against the latest versions of the affected browsers.
now for mobile devices, this property work for touch events, tbh, what I tested is that when I set pointer-events to none I can interact with the dev beneath the div with pointer-events: none.
I'm not sure if this is enough to assume this feature works in the specified browsers.

File: css/properties/pointer-events.json [hunk 1]
```diff
@@ -80,13 +80,13 @@
...
               "opera_android": {
-                "version_added": null
+                "version_added": true
               },
               "safari": {
                 "version_added": "4"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "12"
```
Comment (ddbeck) [inline css/properties/pointer-events.json]: One more please!

```suggestion
                "version_added": true
```

**Comment by Ahmad-Abo-Saa** [comment]
@ddbeck @chankruze safari_ios updated

**Pull request merged**